kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/istio-fix-domain.py

```diff
@@ -0,0 +1,131 @@
+set('ClientCertificateEnforced', false)
+
+cd('/Servers/' + admin_server_name)
+create('istio-t3', 'NetworkAccessPoint')
+cd('/Servers/%s/NetworkAccessPoints/istio-t3' % admin_server_name)
+set('ListenAddress', '127.0.0.1')
+set('PublicAddress', domain_uid + '-' + admin_server_name)
+set('ListenPort', admin_port)
+set('TunnelingEnabled', false)
+set('OutboundEnabled', false)
+set('Enabled', true)
+set('TwoWaySslEnabled', false)
+set('ClientCertificateEnforced', false)
+
+cd('/Servers/' + admin_server_name)
+create('istio-ldap', 'NetworkAccessPoint')
+cd('/Servers/%s/NetworkAccessPoints/istio-ldap' % admin_server_name)
+set('Protocol', "ldap")
+set('ListenAddress', '127.0.0.1')
+set('PublicAddress', domain_uid + '-' + admin_server_name)
+set('ListenPort', admin_port)
+set('HttpEnabledForThisProtocol', true)
+set('TunnelingEnabled', false)
+set('OutboundEnabled', false)
+set('Enabled', true)
+set('TwoWaySslEnabled', false)
+set('ClientCertificateEnforced', false)
+
+cd('/Servers/%s/NetworkAccessPoints/T3Channel' % admin_server_name)
+set('ListenAddress', '127.0.0.1')
+set('PublicAddress', domain_uid + '-' + admin_server_name)
+
+templateName = cluster_name + "-template"
+cd('/ServerTemplates/%s' % templateName)
+create('istio-probe', 'NetworkAccessPoint')
+cd('/ServerTemplates/%s/NetworkAccessPoints/istio-probe' % templateName)
+set('Protocol', 'http')
+set('ListenAddress', '127.0.0.1')
+set('PublicAddress', domain_uid + '-' + managed_server_name_base + '${id}')
+set('ListenPort', 8888)
+set('HttpEnabledForThisProtocol', true)
+set('TunnelingEnabled', false)
+set('OutboundEnabled', false)
+set('Enabled', true)
+set('TwoWaySslEnabled', false)
+set('ClientCertificateEnforced', false)
+
+cd('/ServerTemplates/%s' % templateName)
+create('istio-t3', 'NetworkAccessPoint')
+cd('/ServerTemplates/%s/NetworkAccessPoints/istio-t3' % templateName)
+set('ListenAddress', '127.0.0.1')
+set('PublicAddress', domain_uid + '-' + managed_server_name_base + '${id}')
+set('ListenPort', server_port)
+set('TunnelingEnabled', false)
+set('OutboundEnabled', false)
+set('Enabled', true)
+set('TwoWaySslEnabled', false)
+set('ClientCertificateEnforced', false)
+
+cd('/ServerTemplates/%s' % templateName)
+create('istio-cluster', 'NetworkAccessPoint')
+cd('/ServerTemplates/%s/NetworkAccessPoints/istio-cluster' % templateName)
+set('Protocol', "CLUSTER-BROADCAST")
+set('ListenAddress', '127.0.0.1')
+set('PublicAddress', domain_uid + '-' + managed_server_name_base + '${id}')
+set('ListenPort', server_port)
+set('TunnelingEnabled', false)
+set('OutboundEnabled', false)
+set('Enabled', true)
+set('TwoWaySslEnabled', false)
+set('ClientCertificateEnforced', false)
+
+cd('/ServerTemplates/%s' % templateName)
+create('istio-http', 'NetworkAccessPoint')
+cd('/ServerTemplates/%s/NetworkAccessPoints/istio-http' % templateName)
+set('Protocol', 'http')
+set('ListenAddress', '127.0.0.1')
+set('PublicAddress', domain_uid + '-' + managed_server_name_base + '${id}')
+set('ListenPort', 31111)
```

Comment:
I am thinking a different port should be used here since 31111 is in the nodePort range and this is not a nodePort.